flocker_bb/builders/flocker.py

```diff
@@ -661,6 +661,19 @@ def getBuilders(slavenames):
                 nextSlave=idleSlave,
                 ))
 
+    # Storage backend builders
+    builders.extend([
+        BuilderConfig(
+            name='flocker-storage-backend-cinder',
```

Comment:
`flocker/functional/rackspace/backend/centos-7`?

Maybe that is too much of a mouthful.

Also, this should have a `builddir` which is the same as the name but with `/` replaced with '-'.